### ISSUE TITLE
Help text does not contain default configuration values

### ISSUE THREAD

**Issue by andy0130tw** [rrthomas/pdfjam#45]
The following output is from `pdfjam --help`:

```
pdfjam is a shell-script front end to the LaTeX 'pdfpages' package (for
which, see http://www.ctan.org/tex-archive/macros/latex/contrib/pdfpages).

Usage: pdfjam [OPTIONS] [--] [FILE1 [SEL1]] [FILE2 [SEL2]]...

[...]
     --suffix STRING
                  Specifies a suffix for output file names, to be used when
                  --outfile is either (a) a directory, or
                                      (b) not specified in a --batch call.
                  A good STRING should be descriptive: for example,
                           --suffix 'rotated'
                  would append the text '-rotated' to the name of the input
                  file in order to make the output file name, as in
                  'myfile-rotated.pdf'.  The STRING must not have zero
                  length.
                  [Default for you at this site: suffix=]
[...]
```

All the "Default for you at this site:" section is missing the default configuration value even if those are set. It is because the `$helptext` variable is placed *before* setting their values up. It is a regression since the order is correct in v2.08.

**Comment by DavidFirth** [comment]
That was clumsy programming (by me).  Hopefully easy to fix.  And in my view it's important to fix this one.  But i am the clumsy programmer, so i should stay clear of the actual code now!

**Comment by andy0130tw** [comment]
@DavidFirth No problem. I tried to move sections around and seemed to have fixed the issue shortly after making this issue. But I haven't read any code, so I am not sure if a PR would be crude.

FYI: https://github.com/andy0130tw/pdfjam/tree/fix-helptext-defaults

**Comment by mgkurtz** [comment]
Fixed in ad3eac5aee99fadb9cb7bc75c6147f3d0de6445e.
